src/components/faqbox/index.js

```diff
@@ -1,26 +1,55 @@
-import React from 'react';
-import styled from 'styled-components'
-;
+import React from "react";
+import styled from "styled-components";
 const Box = styled.div`
-    background-color: white;
-    box-shadow: 2px 10px 30px rgba(0, 0, 0, 0.25);
-    border-radius: 7px;
-    text-align: left;
-    width: 20rem;
-    margin: 5rem;
-    padding: 2rem;
+  background-color: white;
+  box-shadow: 2px 10px 30px rgba(0, 0, 0, 0.25);
+  border-radius: 7px;
+  text-align: left;
```

Comment:
Can remove

Reply:
Remove 5-7? I thought that box-shadow and border-radius were a necessary part of the design?

Reply:
oh idk why this highlighted 4 lines I meant just the text-align: left;